docs/02-help-desk/hardware-offerings.md

```diff
@@ -0,0 +1,78 @@
+
+# Requesting hardware or accessories
+
+## Services offered
+
+Bixal offers several different laptop configurations depending on role and job
+requirements. Beyond a laptop Bixal will provide a budget for accessories
+and offerers a list of accessories that can be requested without
+additional input from management.
+
+### A Note on Americans with Disabilities Act
+
+The equipment we purchase and distribute to employees works well for most
+employees. However Bixal understands that accommodations for health reasons
+should be made. If you feel like the standard equipment Bixal distributes
+will have a negative effect, or create challenges for you working at Bixal
+you should discuss that with People and Culture, your project manager,
+or create a ticket at [https://help.bixal.com](https://help.bixal.com)
+to request equipment that will allow you work and communicate with your
+peers without creating impedance.
+
+### Laptop Variants
+
+* Mac Book Pro 16"
+  * Insert specs here.
+* Mac Book Pro 14"
+  * Insert specs here.
+* Generic 15" Windows Based Laptop
+  * Insert specs here.
+* Generic 15" Linux Based Laptop
+  * Insert specs here.
+
+### Accessories
+
+Bixal will allow purchases from the following productivity purchase
+assuming you have availability from your accessory budget.
+
+* Mouse and track pads
+  * Insert current models
+* Keyboards and input
+  * Insert models here
+* Earbuds and Headphones
+
+### Roles and Default Hardware
+
+Most roles at bixal fall into one high level descriptor(Developer,
+Sysadmin, Project Manager, Quality Assurance tester, etc). There
+is a default hardware package for most of these roles. During
+the hiring process People and Culture will reach out to the hiring
+manager to identify the role they are fulfilling. If the hiring manager,
+or hired employee has concerns about the assigned hardware package
+they may follow the steps below to request a different package.
+
+* Developer Approved Packages
+  * MacBook Pro 16" (Default)
+  * Generic 15" Linux Based Laptop
+  * Generic 15" Windows Based Laptop
+* Systems Administrator
+  * Generic 15" Linux Based Laptop
+* Quality Assurance
+  * Generic 15" Windows Based Laptop
+* Project Manager
+  * Mac Book Pro 14"
+* Designer
```

Comment:
@pglevy Wondering if you can give me some guidance here on the different design roles. What are the different roles at a high level, and does the hardware needed matter. Should I reach out to Amanda as well to get some input on video editors?

Reply:
I'll check in with team and get back to you.